utils/hct/hctbuild.cmd

```diff
@@ -346,7 +353,7 @@ if "%BUILD_GENERATOR%" NEQ "Ninja" (
 )
 
 rem Just defer to cmake for now.
-cmake --build . --config %1
+cmake --build . --config %1 -- %PARALLEL_OPT%
```

Comment:
Curious, usually `--` means "don't interpret the rest of the command line, just pass it on", but pass it on to what in this case? The compiler? Linker? Everything?

Reply:
CMake docs on `--build` say "-- Pass remaining options to the native tool.", so anything it calls on `--build`, usually a compiler.

Reply:
Oh, probably MSBuild in this case. Makes sense.